src/d3a/models/strategy/area_agents/two_sided_pay_as_bid_engine.py

```diff
@@ -40,6 +40,11 @@ def _forward_bid(self, bid):
             return
         if self.owner.name == self.markets.target.area.name:
             return
+        if bid.price < (self.markets.target.transfer_fee_const * bid.energy):
```

Comment:
You need to also take into account the transfer_fee_pct here as well. Keep in mind that the end result in this check is to not have a negative price when forwarding the bid.

Reply:
Yes, you are right! Will change this